src/main/java/com/epam/Car.java

```diff
@@ -0,0 +1,56 @@
+package com.epam;
+
+import org.apache.log4j.Logger;
+
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+public class Car implements Runnable {
+
+    private static final long MAX_DISTANCE = 10000;
+    private static AtomicBoolean winnerExists = new AtomicBoolean(false);
+    private static AtomicBoolean interruptedExists = new AtomicBoolean(false);
+    private static String winner;
+
+
+    Logger log = Logger.getLogger(Car.class);
```

Comment:
It would be better to use private static final for Logger. Please read why it should be used for that)